### DISCUSSION TITLE
Third party partitioning in Google Chrome breaking Power BI authentication for embedded dashboards

### DISCUSSION THREAD

**Discussion by ghost** [privacysandbox/privacy-sandbox-dev-support#164] [Phasing out third-party cookies]
Dear experts,

In our company (38.000 employees), Power BI secure embedded dashboards ( Power BI embed for your organization aka user owns data ) fail to complete the authentication process since last week.

This affects our clients and employees severely.

After much investigation we discovered that the flag value 'disable' for the chrome://flag "Experimental third-party storage partitioning" resolves the issue for us. We also observe that the flag value 'default" affects some embedded dashboards, but not all of them. We observe that the flag value "enable" affects all embedded dashboards.

We do not experience these issues in the web browsers Edge or Firefox.
We struggle to find the right place to report this issue. I hope this this the right place. And if not, I hope that someone here would be so kind to point us to the right place, or send this message to the appropriate team responsible for the mentioned technology. We would be very thankfull for any response.

Kind regards,
Thijs the Vries, on behalf of the Savills web development and BI teams

Google Chrome | 117.0.5938.150 (Official Build) (64-bit) (cohort: Stable)
Revision | e3344ddefa12e60436fa28c81cf207c1afb4d0a9-refs/branch-heads/5938@{#1539}
OS | Windows 11 Version 22H2 (Build 22621.2361)
JavaScript | V8 11.7.439.21
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36
Command Line | "C:\Program Files\Google\Chrome\Application\chrome.exe" --flag-switches-begin --disable-features=ThirdPartyStoragePartitioning --flag-switches-end --origin-trial-disabled-features=WebGPU --disable-nacl

**Comment by krgovind** [comment]
Duplicate of #163. We will respond there.
